cmd/arc_summary/arc_summary3.py

```diff
@@ -109,9 +109,14 @@ def draw_graph(kstats_dict):
     arc_perc = f_perc(arc_stats['size'], arc_stats['c_max'])
     mfu_size = f_bytes(arc_stats['mfu_size'])
     mru_size = f_bytes(arc_stats['mru_size'])
-
-    info_form = 'ARC: {0} ({1})  MFU: {2}  MRU: {3}'
-    info_line = info_form.format(arc_size, arc_perc, mfu_size, mru_size)
+    meta_limit = f_bytes(arc_stats['arc_meta_limit'])
+    meta_size = f_bytes(arc_stats['arc_meta_used'])
+    dnode_limit = f_bytes(arc_stats['arc_dnode_limit'])
+    dnode_size = f_bytes(arc_stats['dnode_size'])
+
+    info_form = 'ARC: {0} ({1})  MFU: {2}  MRU: {3}  META: {4} ({5}) DNODE {6} ({7})'
+    info_line = info_form.format(arc_size, arc_perc, mfu_size, mru_size,
+                                 meta_size, meta_limit, dnode_size, dnode_limit)
```

Comment:
@rincebrain whoops, we accidentally let in a style issue.  Would you mind opening a new PR to resolve this.

```
./cmd/arc_summary/arc_summary3.py:117:80: E501 line too long (85 > 79 characters)
./cmd/arc_summary/arc_summary3.py:119:80: E501 line too long (80 > 79 characters)
```
How about also moving "META" and "DNODE" on to the next line, and adding the missing ":" after DNODE.  Something like this.  Sorry, I missed it the first time!

```
    ARC: 4.2 GiB (13.4 %)  MFU: 3.4 GiB  MRU: 321.6 MiB
    META: 1.0 GiB (23.6 GiB) DNODE: 259.4 MiB (2.4 GiB)
    +----------------------------------------------------------+
    |FFFFFF                                                    |
    +----------------------------------------------------------+
```

Reply:
Can do; sorry about that.

Reply:
Why don't we simply remove it for now.  Then we can add a new graph at our leisure if you find it to be useful.